properties/P000133.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: P000133
+slug: linearly_orderable
+name: Linearly orderable
+aliases:
+  - Linearly ordered topological space
```

Comment:
A little long for an alias?  Was thinking more of having this in the description below as an explanation for LOTS. But up to you.